build.sbt

```diff
@@ -8,31 +8,43 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 
 lazy val appName = "api-platform-test-user"
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
+lazy val akkaVersion = "2.5.23"
+lazy val akkaHttpVersion = "10.0.15"
 
 lazy val compile = Seq(
-  "uk.gov.hmrc" %% "bootstrap-play-25" % "5.1.0",
-  "uk.gov.hmrc" %% "play-ui" % "8.3.0-play-25",
-  "uk.gov.hmrc" %% "play-hmrc-api" % "3.4.0-play-25",
+  "uk.gov.hmrc" %% "bootstrap-play-26" % "1.4.0",
```

Comment:
I think many of the play libs are transitive deps of bootstrap

Reply:
play-hmrc-api appears to be
play-ui isn't